day6/salestax.py

```diff
@@ -1,33 +1,68 @@
-import csv
-import pandas
 import fs 
+import csv
+import json
+
+import pandas as pd
```

Comment:
Don't use pandas at all as for now our goal is to learn that basic python features i.e you should know how to write a CSV with python.